build-info-extractor-gradle/src/test/java/org/jfrog/gradle/plugin/artifactory/GradlePluginTest.java

```diff
@@ -59,6 +60,9 @@ public void configurationsTest(String gradleVersion) throws IOException {
         BuildResult buildResult = runGradle(gradleVersion, envVars, false);
         // Check results
         checkBuildResults(artifactoryManager, buildResult, false, localRepo1);
+        // Cleanup
+        Build buildInfo = getBuildInfo(artifactoryManager, buildResult);
```

Comment:
This REST is redundant. You already have the build name and the build number in the `buildResult`.
Also - shouldn't we make the build number random?

Reply:
By default, BuildInfo is published with a build number that is the start date of the build.